generator/build.gradle.kts

```diff
@@ -0,0 +1,12 @@
+import com.expediagroup.sdk.xap.generator.mustache.AllowedMediaTypesLambda
```

Comment:
This import isn't available in this PR. Let's leave it out till later..

Reply:
Done.